invenio_pages/records/models.py

```diff
@@ -36,6 +37,9 @@ class PageModel(db.Model, Timestamp):
     title = db.Column(db.String(200), nullable=False, default="")
     """Page title."""
 
+    lang = db.Column(db.String(2), nullable=False, default="en")
```

Comment:
```suggestion
    lang = db.Column(db.CHAR(2), nullable=False, default="en")
```

minor: very minor optimization, since String becomes `varchar`, and we know that it's a controlled value.

Reply:
Shouldn't this then be changed in the alembic migration script, too?